html/js/tree/Node.js

```diff
@@ -0,0 +1,94 @@
+GAMBIT.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new Node.
+    * @class
+    * @param {Node} [father] Father node. If null, this is root.
+    */
+    function Node(father) {
+        this.father = father;
```

Comment:
Can we use parent instead of father? Mostly for consistency with older gte, so its easier to port stuff that uses the parent property there.

Also see: http://en.wikipedia.org/wiki/Tree_structure#Terminology_and_properties